### DISCUSSION TITLE
Navigation Line customisation

### DISCUSSION THREAD

**Discussion by k-koleda** [organicmaps/organicmaps#4134] [Ideas]
On some roads blue navigation line is so bold. often it can be difficult to understand which exit you need especially on road with 5-6 exit on round bound. would be great to have setting where I can customise navigation line to make the line narrower or the same size as road, but no wider than the road. attached an example. so I mean navigation line is wide and need a time to understand which exit I need.
![2022-12-22 13 33 45](https://user-images.githubusercontent.com/32541582/209135506-74893902-ab38-4245-8724-7843b4f67a7a.jpg)

**Comment by biodranik** [comment]
A proper solution won't require any customization. The width should take into account the underlying road's width either dynamically, or it can be reduced globally.
